src/docs/09_design_decisions.adoc

```diff
@@ -4,7 +4,12 @@
 === Route specification
 We follow the https://github.com/Arquisoft/viadeSpec[Viade Specficiation] to set a format to manage the routes, so that interoperability can be achieved.
 
-
 === Leaflet
 We chose this library because it is open source, free, and matches the philosophy of Solid.
 
+=== Template
+At the begining of the course, we had to choose using the Inrupt Generator or not. After some discussion, we decided not to use it in order to learn React from the basics, as we considered not only important the SOLID challenge, but also learning a new technology.
```

Comment:
I will maybe rephrase that and say something like: _we decided not to use it in order to learn React from the basics, because the main goal of this subject is learning how to design a software arquitecture, and with a template extracted from the internet you are force to use the one that is provided._ Just to remark our use of the plain framework.